lib/listeners/win.js

```diff
@@ -37,7 +59,34 @@ exports.callback = function (route, message, response) {
   });
 };
 
-function saveGame(winner, loser) {
+function postRankingToLeaderboard (rankings) {
```

Comment:
So the problem with this is if other people are using it, you'll get a bunch of other garbage data.

One alternative if you want to have access to that data is #24 and using mongo (or whatever data store you want) to get to it. Would that work?